src/config/index.ts

```diff
@@ -0,0 +1,12 @@
+const ROOT_NODE = document.getElementById('root')
```

Comment:
I centralized all configuration I could find in this file. Just a preference of mine so if there are objections I'll revert